Packs/CortexXDR/Integrations/XQLQueryingEngine/XQLQueryingEngine.py

```diff
@@ -785,8 +793,10 @@ def main() -> None:
     """
     args = demisto.args()
     params = demisto.params()
-    api_key = params.get('apikey')
-    api_key_id = params.get('apikey_id')
+    if not params.get('apikey') or not (api_key := params.get('apikey', {}).get('password')):
```

Comment:
Why are you using the `:=` operator here?

Reply:
Fixed